ui/src/components/spinner/Spinner.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Spinner = ({ status }) => {
+    const spinnerStyle = {
+        boxSizing: "border-box",
+        margin: "auto",
+        width: "120px",
+        height: "120px",
+        borderRadius: "50%",
```

Comment:
Can we move this style into App.scss too? We generally want to avoid using 'CSS in JSX' when we are using local SCSS files, it just means aren't using different methods to apply our style and keep consistent